Source/qol/stash.cpp

```diff
@@ -494,53 +449,62 @@ bool UseStashItem(uint16_t c)
 
 	UseItem(MyPlayerId, item->_iMiscId, item->_iSpell);
 
-	if (Stash.stashList[c]._iMiscId == IMISC_MAPOFDOOM)
+	if (Stash.stashList[itemIndex]._iMiscId == IMISC_MAPOFDOOM)
 		return true;
-	if (Stash.stashList[c]._iMiscId == IMISC_NOTE) {
+	if (Stash.stashList[itemIndex]._iMiscId == IMISC_NOTE) {

```

Comment:
might as well use the `Item &item` alias here

Reply:
Suggestion applied in most recent push